core/src/main/scala/org/dbpedia/extraction/mappings/GeoCoordinatesMapping.scala

```diff
@@ -29,83 +28,76 @@ class GeoCoordinatesMapping(
   val latitudeDirection : String,
   context : {
     def ontology : Ontology
-    def redirects : Redirects   // redirects required by GeoCoordinatesParser
-    def language : Language 
-  } 
-) 
-extends PropertyMapping
-{
+    def redirects : Redirects
+    def language : Language
+  }
+) extends PropertyMapping {
+
   private val logger = Logger.getLogger(classOf[GeoCoordinatesMapping].getName)
 
   private val geoCoordinateParser = new GeoCoordinateParser(context)
   private val singleGeoCoordinateParser = new SingleGeoCoordinateParser(context)
   private val doubleParser = new DoubleParser(context)
-  private val doubleParserEn = new DoubleParser(context = new {def language : Language = Language("en")})
+  private val doubleParserEn = new DoubleParser(context = new { def language : Language = Language("en") })
   private val stringParser = StringParser
   private val wikiCode = context.language.wikiCode
 
   private val typeOntProperty = context.ontology.properties("rdf:type")
   private val latOntProperty = context.ontology.properties("geo:lat")
   private val lonOntProperty = context.ontology.properties("geo:long")
   private val pointOntProperty = context.ontology.properties("georss:point")
-  private val featureOntClass =  context.ontology.classes("geo:SpatialThing")
+  private val featureOntClass = context.ontology.classes("geo:SpatialThing")
+
+  // ✅ NEW hemisphere properties
+  private val latnsOntProperty = context.ontology.properties("dbp:latns")
```

Comment:
I think we discussed that we will be trying to convert hemisphere into the correct sign on the computed lat/lon values, so there will not be needed here